pkg/controller/dbsyncer/intervalpolicy/syncer_interval_policy.go

```diff
@@ -0,0 +1,10 @@
+package intervalpolicy
+
+import "time"
+
+// SyncerIntervalPolicy defines a policy to return syncer interval based on the received events.
+type SyncerIntervalPolicy interface {
```

Comment:
@OlegSternbergIBM I thought about it a bit more, maybe instead of `intervalPolicy` a better name would be `backoffPolicy`. what do you think?

Reply:
I think interface name shouldn't expose the actual implementation name